docs/standard/microservices-architecture/multi-container-microservice-net-applications/implementing-api-gateways-with-ocelot.md

```diff
@@ -0,0 +1,566 @@
+---
+title: Implementing API Gateways with Ocelot
+description: .NET Microservices Architecture for Containerized .NET Applications | Implementing API Gateways with Ocelot, an Open Source lightweight API Gateway based on .NET Core
```

Comment:
Let's start adding proper descriptions. The description is a call to action for someone to visit your page, it should tell the reader what they're going to get from this page.

Reply:
👍